static/app/views/replays/replays.tsx

```diff
@@ -105,7 +115,7 @@ class Replays extends React.Component<Props> {
                 <PanelTable
                   isLoading={data.isLoading}
                   isEmpty={data.tableData?.data.length === 0}
-                  headers={['Replay ID', 'User', 'Timestamp']}
+                  headers={['Session', 'Timestamp']}
```

Comment:
Oh one thing I missed, if you could make a follow-up PR for this to use `t()` from `app/locale` -- we use it for i18n. (In this basic case, we can just replace these with `t('Session')`, etc.

Reply:
Sure!